libsolidity/parsing/Parser.cpp

```diff
@@ -687,6 +688,9 @@ ASTPointer<VariableDeclaration> Parser::parseVariableDeclaration(
 			nodeFactory.setEndPositionFromNode(type);
 	}
 
+//	if (!_options.isStateVariable && documentation != nullptr)
```

Comment:
todo

Reply:
i think it make sense to not allow natspecs on local variables